lib/oelite/baker.py

```diff
@@ -76,9 +76,12 @@ def add_bake_parser_options(parser):
 
 
 def add_show_parser_options(parser):
+    import optparse
     parser.add_option("--nohash",
                       action="store_true",
                       help="don't show variables that will be ignored when computing data hash")
+    parser.add_option("--nodynvar",
+                      action="store_true", help=optparse.SUPPRESS_HELP)
```

Comment:
I don't think it should be hidden, but I also fail to understand how this is useful. It seems to me that you are being confused about the output showing "not fully expanded" stuff, but are implementing a handle to expand even less?

suggestion for help text:
help="Do not show the expanded value of the special 'dynamic variables' which are values the oe-lite generates based on e.g. git checkout information"